### PR TITLE
Feat: Quiz-App Pages added

### PR DESCRIPTION
**Pull request by mdarif6**
- Quiz-App Pages added
- Questions Added
- Question selection and click feature added

### CODE REVIEW

File: src/common/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import "./Header.css";
+import logo from "../assets/images/logo1.PNG";
+import { Link } from "react-router-dom";
+export default function Header() {
+  return (
+    <div className="sm-header">
+      <div className="sm-logo-and-heading">
+        <div className="sm-logo">
+          <Link to="/">
+            <img src={logo} alt="" />
```
Comment (ravikumar1002) [inline src/common/Header.jsx]: fill alt attribute

File: src/context/quiz-context.jsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
+import { createContext, useContext, useReducer } from "react";
+
+const QuizContext = createContext();
+
+const initialState = {
+  quizData1: [
```
Comment (ravikumar1002) [inline src/context/quiz-context.jsx:6]: just a suggestion:  you can make a separate file for question's data

**Review by hrishi82 — APPROVED** [review]
Code looks good to me. 
Suggesetion: For review, You can ignore the package JSON files

File: src/home-page/HomeMain.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+}
+.sm-categ-card {
+  background-color: black;
+  border-radius: 10px;
+}
+.sm-card-details {
+  color: white;
+  padding: 1rem 1rem 0 1rem;
+}
+/* .btn-primary {
```
Comment (ravikumar1002) [inline src/home-page/HomeMain.css]: if it not necessary then you can remove this

**Review by ravikumar1002 — APPROVED** [review]
everything looks  good , 
nice naming convention,
good to merge

File: src/common/Header.jsx (hunk 1 above)
Comment (mdarif6) [reply src/common/Header.jsx]: done

File: src/context/quiz-context.jsx (hunk 2 above)
Comment (mdarif6) [reply src/context/quiz-context.jsx:6]: Will see this

File: src/home-page/HomeMain.css (hunk 3 above)
Comment (mdarif6) [reply src/home-page/HomeMain.css]: done

**Pull request merged**